the-collab-lab/website-og
# Build the site with 11ty

# Description

Start using 11ty to build TCL site.

# What's pending?

- [x] Solve the merge conflicts with `main`

# How do I test this thing?

```
git checkout next
npm install
npm run dev
```
Open http://localhost:8080 and start clicking around. Everything should be working exactly as the site we have in the `main` branch.

# How do get the production version of the site?

```
git checkout next
npm install
npm run build:serve
```

Then, click the link that's suggested by `serve` and click around the site.

## Update

@segdeha this is ready to roll now.

- [ ] Everything looks OK and the links are working as expected
- [ ] Merge this into `main`
- [ ] Update Netlify to run `npm run build` before deploying the site. Everything else should work just fine as 11ty outputs the site to the `/build` folder.

## Loom
https://www.loom.com/share/953cb6c8ca5d4983bb59579cc9d9f44c
